src/components/presentational/ViewHeader/ViewHeader.css

```diff
@@ -15,4 +16,12 @@
     font-weight: 500;
     color: var(--mdhui-text-color-2);
     font-size: 1.2em;
+}
+
+.mdhui-view-header .mdhui-view-header-action-icon {
+    height: 48px;
+    width: 48px;
+    line-height: 48px;
+    font-size: 34px;
+    text-align: center;
```

Comment:
Wouldn't you want text align right here?

Reply:
It was center aligned because the touch target was 48x48 and the icon was 34x34.  I wanted the icon to be centered in the touch target.  However, this CSS class has been removed now that we are using a ReactNode instead.